### PR TITLE
Fixes(#7561) Add support for postgres.UUID literal_binds compilation

### PR DESCRIPTION
**Pull request by jmg-duarte**
### Description
- Add `literal_processor` to `postgresql.UUID`
- Add respective test

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [X] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

**Comment by jmg-duarte** [comment]
@zzzeek This is currently a WIP, I'm having trouble understanding how to run the tests, I don't understand how to set them up and there isn't a guide? Maybe I missed it but it really isn't clear from https://www.sqlalchemy.org/develop.html

Regardless, I think this is kind of what you said and I'd be glad to take some pointers to finish this PR

**Comment by CaselIT** [comment]
> I'm having trouble understanding how to run the tests,

you can refer to https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst

File: test/dialect/postgresql/test_types.py [hunk 1]
```diff
@@ -1420,9 +1421,11 @@ def test_array_agg_generic(self):
         argnames="with_enum, using_aggregate_order_by",
     )
     def test_array_agg_specific(self, with_enum, using_aggregate_order_by):
-        from sqlalchemy.dialects.postgresql import aggregate_order_by
-        from sqlalchemy.dialects.postgresql import array_agg
-        from sqlalchemy.dialects.postgresql import ENUM
+        from sqlalchemy.dialects.postgresql import (
+            ENUM,
+            aggregate_order_by,
+            array_agg,
```
Comment (CaselIT) [inline test/dialect/postgresql/test_types.py:1427]: just fyi we use zimports that formats the imports as before

Comment (jmg-duarte) [reply test/dialect/postgresql/test_types.py:1427]: I ran it and it left it that way. Both like `zimport <changed-file>` and with `pre-commit`.
Did I miss anything?

**Comment by jmg-duarte** [comment]
> > I'm having trouble understanding how to run the tests,
> 
> you can refer to https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst

I read the guides but didn't saw that 🤦 thank you!

PS: Maybe we should add a quick reference to the file somewhere? Or did I miss the reference too?

File: test/dialect/postgresql/test_types.py (hunk 1 above)
Comment (CaselIT) [reply test/dialect/postgresql/test_types.py:1427]: Probably it does not support imports inside the functions, so it skips these.

**Comment by CaselIT** [comment]
> > > I'm having trouble understanding how to run the tests,
> > 
> > you can refer to https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst
> 
> I read the guides but didn't saw that 🤦 thank you!
> 
> PS: Maybe we should add a quick reference to the file somewhere? Or did I miss the reference too?

Is it not mentioned in the develop page on the website?

File: test/dialect/postgresql/test_types.py (hunk 1 above)
Comment (jvanasco) [reply test/dialect/postgresql/test_types.py:1427]: @CaselIT IIRC, zimports doesn't touch imports in functions, try/except blocks, or if/else statements. I think it just looks for "zero indented" imports.

**Comment by jmg-duarte** [comment]
> > > > I'm having trouble understanding how to run the tests,
> > > 
> > > 
> > > you can refer to https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst
> > 
> > 
> > I read the guides but didn't saw that 🤦 thank you!
> > PS: Maybe we should add a quick reference to the file somewhere? Or did I miss the reference too?
> 
> Is it not mentioned in the develop page on the website?

Re-read it, it is there. My bad

**Comment by CaselIT** [comment]
well let's run in in the pipeline to see what's up

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision 590ef4199f4c7b581c47dfff538c4bc5756af43a of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change 590ef4199f4c7b581c47dfff538c4bc5756af43a: https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3503

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision cf6fe73265342d7884a940c4b3a34c9552113ec3 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
Patchset cf6fe73265342d7884a940c4b3a34c9552113ec3 added to existing Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3503

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3503 has been **merged**. Congratulations! :)

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3523 has been **merged**. Congratulations! :)

**Pull request closed without merging**